certbot-nginx/certbot_nginx/tls_sni_01.py

```diff
@@ -141,7 +140,11 @@ def _make_server_block(self, achall, addrs):
         document_root = os.path.join(
             self.configurator.config.work_dir, "tls_sni_01_page")
 
-        block = [['listen', ' ', str(addr)] for addr in addrs]
+        no_default_addrs = [copy.deepcopy(addr) for addr in addrs]
```

Comment:
It seemed unpythonic to have a separate to_string method... I suppose JSON has this though.